active_directory_201.rst

```diff
@@ -20,10 +20,81 @@ DHCP
 Global Catalog
 --------------
 
+The Global Catalog (GC) holds a subset of attributes for all objects in a multi-domain forest.
+It can be used to search all objects in the forest.
+For example, when searching for the a specific user, the GC can be searched to find users not only in the current domain but also in other domains in the forest.
+The attributes included in the GC's subset of object attributes are those required by the schema and those most commonly used in searches.
+Attributes can be added and removed from the Global Catalog using the Active Directory Schema snap-in.
 
-Advanced Active Directory Maintenance
-=====================================
+When Active Directory Domain Services is installed,  the GC is added to the first domain controller in the forest.
+It can be added to additional domain controllers.
+In order to avoid traversing a WAN to gather GC information, it is recommended that each Active Directory site have at least one Global Catalog server.
+
+Universal groups can contain objects from any domain in the forest.
+The GC holds the membership for all universal groups.
+As part of the authorization process, the authenticating DC retrieves the security identifier for all the member's security groups, which includes universal groups.
+Therefore, if the environment has more than one domain, users need to be able to access a global catalog server for authentication in order to find objects from the universal groups in other domains.
 
 FSMO Roles
 ----------
 
+FSMO stands for Flexible Single Master Operation.
+FSMO roles are also referred to as Operations Master roles.
+They are various functions performed by Domain Controllers (DCs) in Active Directory.
+
+There are 5 FSMO roles.
+Three of the roles exist for every domain in the forest and two apply to the entire forest.
+While it is possible to have 5 domain controllers each with a separate FSMO role, it is common to have multiple roles on one server.
+The forest-wide roles are Schema Master and Domain Naming Master.
+The domain-wide roles are PDC Emulator, RID Master, and Infrastructure Master.
+
+Schema Master
+^^^^^^^^^^^^^
+
+This role is forest-wide.
+This domain controller can update the schema.
+Given that the schema is not frequently updated, availability of this DC is less important than it is for other roles.
+By default, the role is held on the first server promoted to a DC in the forest.
+
+Domain Naming Master
+^^^^^^^^^^^^^^^^^^^^
+
+This role is also forest-wide.
+It controls changes to the forest namespace.
+It can add, remove, rename, and move domains.
+Since domain names changes are infrequent, this role is rarely used.
+By default, the role is held on the first server promoted to a DC in the forest.
+
+PDC Emulator
```

Comment:
This section explains a lot about PDC, but not why there's an Emulator. Can you add something to explain that?